app/LoginPage/page.tsx

```
@@ -0,0 +1,92 @@
+"use client";
+
+import { useState } from "react";
+import { Button } from "../components/ui/buttons";
+import { Input } from "../components/ui/input";
+import Image from "next/image";
+import { Icon, Mail } from "lucide-react";
+
+export default function LoginPage() {
+  const [email, setEmail] = useState("");
```

ESLint

[error] 'email' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)